lib/src/utils/serialization_utils.dart

```diff
@@ -891,9 +896,11 @@ class SerializationUtils {
   }
 
   static Uint8List int64ToBuf(BigInt value) {
-    if (value.isNegative) {
-      throw ValidationException("int64ToBuf - Value is negative");
-    } else if (value.bitLength > 64) {
+    //TODO: Check if this is ok because there are cases where values could be negative
+    // if (value.isNegative) {
+    //   throw ValidationException("int64ToBuf - Value is negative");
+    // }
+    if (value.bitLength > 64) {
       throw ValidationException("int64ToBuf - Value is too large");
     }
```

Comment:
It's normal to accept negative values for this. It's int64 not uint64. However, max bitLength should change to `63` instead of `64` because one bit is reserved for sign.

Reply:
done